android/lib/build.gradle

```diff
@@ -16,7 +16,7 @@ android {
 }
 
 dependencies {
-  compile 'com.facebook.react:react-native:0.29.2'
-  compile "com.google.android.gms:play-services-base:9.2.0"
-  compile 'com.google.android.gms:play-services-maps:9.2.0'
+  compile 'com.facebook.react:react-native:+'
+  compile "com.google.android.gms:play-services-base:8.4.0"
+  compile 'com.google.android.gms:play-services-maps:8.4.0'
```

Comment:
As far as outdated play services is concerned. I think I checked and I couldn't find how you can detect what play services version the phone is currently on.

If this is possible, I would suggest we do a check against whatever version this maps library requires. If the user has an outdated version we emit an event over the bridge so a proper alert/action can be given to the user.